data/json/start_locations.json

```diff
@@ -5,6 +5,96 @@
     "name": "Shelter",
     "target": "shelter"
   },
+  {
+      "type": "start_location",
+      "ident": "road_end_south",
+      "name": "Road End (South)",
+      "target": "road_end_south"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_end_west",
+      "name": "Road End (West)",
+      "target": "road_end_west"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_ne",
+      "name": "Road (NE)",
+      "target": "road_ne"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_end_north",
+      "name": "Road End (North)",
+      "target": "road_end_north"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_ns",
+      "name": "Road (NS)",
+      "target": "road_ns"
+  },  
+   {
+      "type": "start_location",
+      "ident": "road_es",
+      "name": "Road (ES)",
+      "target": "road_es"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_nes",
+      "name": "Road (NES)",
+      "target": "road_nes"
+  },
+  {
+      "type": "start_location",
+      "ident": "road_end_east"
+      "name": "Road End (East)"
+      "target": "road_end_east"
```

Comment:
Missed this one

Reply:
Done.

Reply:
Sorry, for the (very) late reply, but that's because you created this on an older version, and those files were updated in between.

Reply:
Can I merge the current version and hope it works? I have a feeling it would erase my changes...

Reply:
There should be a button where it says what files are conflicting that lets you fix it.

Reply:
The button is locked. Odd. 

"only these with write access to this repository can merge pull requests"

I have write access to my own repository, right? Then why won't it let me fix it?

Reply:
There should be a different button, somewhere in that area. It might be a link, it's been a while since I've used to website for merge conflicts.

Reply:
"These conflicts are too complex to resolve in the web editor"

I'll be downloading the desktop version then.

Reply:
> There should be a different button, somewhere in that area. It might be a link, it's been a while since I've used to website for merge conflicts.


[Here's what I saw.](https://imgur.com/lbOPK91)

Reply:
Yeah, the greyed out 'Resolve conflicts` button was what I was referring to.

Reply:
What do I do now? Do I have to delete all my work and restart from scratch?

Reply:
No, you could resolve conflicts manually.

Reply:
Alright, how can I do that? It'd be much appreciated.

Reply:
https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line

Reply:
There was nothing out of the ordinary when I looked at `start_locations.json` in Atom. No `<<<<HEAD>>>` or anything. 

[start_locations file](https://imgur.com/a/hYUfgte.png)


Looked at `data/json/scenarios/scenarios.json`, nothing there either.

[scenarios.json file 1](https://imgur.com/a/QgtEI5J.png)

[scenarios.json file 2](https://imgur.com/a/HpZ13TZ.png)


And here at mapgen_extra.ccp:

[mapgen_extra.ccp](https://imgur.com/a/CPlCT2F.png)

I'll also upload a pic of git status.